ick/rules/docker.py

```diff
@@ -2,18 +2,19 @@
 import shlex
 
 from ..base_rule import BaseRule, ExecWork
+from ..config import RuleConfig
 from ..sh import run_cmd
 
 
 class Rule(BaseRule):
     work_cls = ExecWork
 
-    def __init__(self, rule_config, repo_config) -> None:  # type: ignore[no-untyped-def] # FIX ME
-        super().__init__(rule_config, repo_config)  # type: ignore[no-untyped-call] # FIX ME
+    def __init__(self, rule_config: RuleConfig) -> None:
+        super().__init__(rule_config)
         if isinstance(self.rule_config.command, str):
             parts = shlex.split(self.rule_config.command)
         else:
```

Comment:
If you need to pick str or list I'd rather keep the list part.  As currently typed you don't need this branch.